tests/fixtures/composerProvider.yml

```diff
@@ -0,0 +1,33 @@
+-
+  input:
+    file: 'composer1.json'
+    artifact: 'dependency-1-{pretty-version}.tar.gz'
+    type: 'tar'
+    command: 'install'
+  output:
+    existing:
+      - /vendor/openeuropa/dependency-1/composer.json
+      - /vendor/openeuropa/dependency-2/composer.json
+      - /vendor/openeuropa/dependency-1/artifact.txt
+-
+  input:
+    file: 'composer2.json'
+    artifact: 'dependency-1-{pretty-version}.zip'
+    type: 'zip'
+    command: 'install'
+  output:
+    existing:
+      - /vendor/openeuropa/dependency-1/composer.json
+      - /vendor/openeuropa/dependency-2/composer.json
+      - /vendor/openeuropa/dependency-1/artifact.txt
+-
+  input:
+    file: 'composer1.json'
+    artifact: 'dependency-1-{pretty-version}.tar.gz'
+    type: 'tar'
+    command: 'install --prefer-source'
+  output:
+    existing:
+      - /vendor/openeuropa/dependency-1/composer.json
+      - /vendor/openeuropa/dependency-2/composer.json
+      - /vendor/openeuropa/dependency-1/artifact.txt
```

Comment:
This test is wrong, artifact should not be existing if launched with `--prefer-source`.

Reply:
Fixed by adding a comment in the .yml file.